WalletWasabi.Fluent/ViewModels/CoinSelection/Core/CommandViewModel.cs

```diff
@@ -0,0 +1,19 @@
+using System.Diagnostics.CodeAnalysis;
+using System.Reactive;
+using ReactiveUI;
+
+namespace WalletWasabi.Fluent.ViewModels.CoinSelection.Core;
+
+[SuppressMessage("ReSharper", "MemberCanBePrivate.Global", Justification = "Used in XAML via Style")]
+[SuppressMessage("ReSharper", "UnusedAutoPropertyAccessor.Global", Justification = "Used in XAML via Style")]
```

Comment:
Not every collaborator has ReSharper, you should never add these kinds of lines.